doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>Maybe add a line explaing what the bucketfs is? link to docu?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Hmm, the user is supposed to be familiar with main Exasol concepts. Plus this is not the starting page of the tutorial. Such references, if required, would be placed earlier.</p>

Reply:
<p>well but not all users are familiar. especially if it is new users they might not now about the bucketfs, as it is not very apparent in the Saas that it even exists. You can also move this to the configuration notebook if you want. but i think it would be really useful for users who are not so familiar with exasol yet to have a direct link. especially since adding the link is not a time intensive task.</p>